### ISSUE TITLE
Rules: Empty Button False Positive

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1892]
On https://hcde.org/choosehamilton/

This code is falsely flagging as a false positive:
```
<a class="link_text" role="button" href="/charter-schools"><span class="text">Discover more...</span><svg aria-hidden="true" width="20" height="20" viewBox="0 0 22 22" preserveAspectRatio="xMidYMid meet">
<g transform="matrix(1,0,0,-1,12,11)">
<g transform="matrix(1,0,0,-1,-1,3)">
<path class="line" stroke-linejoin="round" stroke-linecap="round" fill-opacity="0" stroke="#000" stroke-width="1.5" d="M 7 3.3 L -3.2 3.3 C -5 3.3 -6.5 1.8 -6.5 0 L -6.5 -2.5"></path>
</g>
<g transform="matrix(1,0,0,-1,4,-0.5)">
<path class="arrow" fill-opacity="0" stroke-linejoin="round" stroke-linecap="round" stroke-width="1.5" stroke="#000" d="M -2.5 5 C -2.5 5 2.5 0 2.5 0"></path>
<path class="arrow" fill-opacity="0" stroke-linejoin="round" stroke-linecap="round" stroke-width="1.5" stroke="#000" d="M -2.5 -5 C -2.5 -5 2.5 0 2.5 0"></path>
</g>
</g>
</svg></a>
```

<img width="1401" height="966" alt="Image" src="https://github.com/user-attachments/assets/cdf63412-4bd3-4ee9-bcc6-9ddcd881e891" />

I listened with a screen reader and confirmed that the Discover More text is read.

Can we please look into what is causing this button to false flag?
